### DISCUSSION TITLE
Error while accessing 1.0.188-alpha.2 (local build)

### DISCUSSION THREAD

**Discussion by dannyshin76** [Budibase/budibase#6171] [Help]
Q1)
After building 1.0.188-alpha.2 for local hosting by `yarn build:docker` and running up the service with docker-compose, when we try to access the UI, we get `Internal Server Error`:
![image](https://user-images.githubusercontent.com/93232582/171516802-0ece0048-1e43-415e-8bca-2c8fd43f1133.png)

On the console, we see `Global DB not found`:
![image](https://user-images.githubusercontent.com/93232582/171516838-75d812cc-e298-47e1-a70a-a78175e0623d.png)

Could you advise on what to do?

Q2)
Also, the built image size is very different from your released image for distribution:
![image](https://user-images.githubusercontent.com/93232582/171517148-2ac523d2-5f21-4c41-aaa5-2f2238da45b8.png)
(For app-service, the local build is 1.22GB, distribution image is 2.13GB)

Is this normal, or is something wrong?

**Comment by shogunpurple** [comment]
@dannyshin76 - that version you are using is a development version of budibase, so it's not a supported/stable version. Is there any reason that you can't use the distributed versions? The latest is v1.0.194 https://github.com/Budibase/budibase/releases

I asked the same question in your other discussion, here: https://github.com/Budibase/budibase/discussions/6122

To answer your question about the images - Is it possible that your local version does not have the NPM modules installed? Did you run:

```
yarn && yarn bootstrap
yarn build
yarn build:docker
```

What OS are you using? ARM or AMD? 

It would be very helpful if you could provide some more information about what you are trying to achieve and why. Thanks

**Reply by dannyshin76** [reply]
We tried again with release v1.0.194 that you have suggested:

- Build was successful by running the following in the specified order:
`yarn`
`yarn bootstrap`
`yarn build`
`yarn build:docker`
- When we tried local access, the log-in screen appeared
- However, right after logging in, `Global DB not found` has occurred:
![image](https://user-images.githubusercontent.com/93232582/171784009-62ab8ecf-a249-4a26-a46e-8bb04586154f.png)

Context:
- npm is installed
- Architecture is AMD64, OS is Linux Mint 20.2
- We need to be able to build a local version because we need some customization. Not sure about the details but it's about domain redirection and some other minor things regarding mySQL DB (not an engineer here :()
- As far as I know, we were successful with release 1.0.105-alpha.8, which we are using now.

Could you advise with this `Global DB not found` problem?

**Comment by actus-wirtenberger** [comment]
We are also running into this issue:

### To Reproduce

1. Clone [06b7442500b1d07af89bdf19b7bcef412fee9afc](https://github.com/Budibase/budibase/commit/06b7442500b1d07af89bdf19b7bcef412fee9afc)
2. Edit `hosting/scripts/release-to-docker-hub.sh`
```
#!/bin/bash

tag=$1

if [[ ! "$tag" ]]; then
	echo "No tag present. You must pass a tag to this script"
	exit 1
fi

echo "Tagging images with tag: $tag"


docker login <<custom registry>>

docker tag proxy-service <<custom registry>>/budibase/proxy:$tag
docker tag app-service <<custom registry>>/budibase/apps:$tag
docker tag worker-service <<custom registry>>/budibase/worker:$tag

docker push --all-tags <<custom registry>>/budibase/apps 
docker push --all-tags <<custom registry>>/budibase/worker
docker push --all-tags <<custom registry>>/budibase/proxy
```
3. Run
```
yarn setup
export BUDIBASE_RELEASE_VERSION=1.0.192-alpha.3
yarn build:docker
```

The resulting docker images have the issues described by @dannyshin76.

The same result on both 1.0.188-alpha.2 as mentioned and on the latest 1.0.192-alpha.3

Architecture is AMD64, OS is Ubuntu 22.04 LTS
